src/identity-providers/add/DetailSettings.tsx

```diff
@@ -95,11 +98,19 @@ export const DetailSettings = () => {
   const { realm } = useRealm();
 
   useFetch(
-    () => adminClient.identityProviders.findOne({ alias: id }),
+    () => {
+      return Promise.all([
+        adminClient.identityProviders.findOne({ alias: id }),
+        adminClient.identityProviders.findMappers({ alias: id }),
+        adminClient.identityProviders.findMapperTypes({ alias: id }),
+      ]);
+    },
     (provider) => {
```

Comment:
Instead of using an index to access the provider you can de-structure the array and get the fist item.
```suggestion
    ([provider]) => {
```